From 0a66446154d95222a9e2a45a2120050842b64257 Mon Sep 17 00:00:00 2001
From: "Philip, Avinash" <avinashphilip@ti.com>
Date: Wed, 18 Jan 2012 11:12:46 +0530
Subject: [PATCH 0718/1092] Backlight - Removed backlight data related to GPIO

As backlight is enabled through PWM, cleaning up the GPIO based
backlight data.

Signed-off-by: Philip, Avinash <avinashphilip@ti.com>
---
 arch/arm/mach-omap2/board-am335xevm.c | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/arch/arm/mach-omap2/board-am335xevm.c b/arch/arm/mach-omap2/board-am335xevm.c
index 7bd438f..2a4da66 100644
--- a/arch/arm/mach-omap2/board-am335xevm.c
+++ b/arch/arm/mach-omap2/board-am335xevm.c
@@ -125,7 +125,6 @@ static const struct display_panel disp_panel = {
 	COLOR_ACTIVE,
 };
 
-#define AM335X_LCD_BL_PIN	GPIO_TO_PIN(0, 7)
 /* LCD backlight platform Data */
 #define AM335X_BACKLIGHT_MAX_BRIGHTNESS        250
 #define AM335X_BACKLIGHT_DEFAULT_BRIGHTNESS    250
@@ -158,13 +157,10 @@ static struct lcd_ctrl_config lcd_cfg = {
 	.raster_order		= 0,
 };
 
-static void am335x_gpio_bl_ctrl(int val);
-
 struct da8xx_lcdc_platform_data TFC_S9700RTWV35TR_01B_pdata = {
 	.manu_name		= "ThreeFive",
 	.controller_data	= &lcd_cfg,
 	.type			= "TFC_S9700RTWV35TR_01B",
-	.panel_power_ctrl	= am335x_gpio_bl_ctrl,
 };
 
 #include "common.h"
@@ -866,11 +862,6 @@ static void enable_ecap0(int evm_id, int profile)
 	backlight_enable = true;
 }
 
-static void am335x_gpio_bl_ctrl(int val)
-{
-	/* lcd backlight */
-	gpio_set_value(AM335X_LCD_BL_PIN, val);
-}
 /* Setup pwm-backlight */
 static struct platform_device am335x_backlight = {
 	.name           = "pwm-backlight",
-- 
1.7.11.2

